packages/vite-plugin-cloudflare/src/plugin-config.ts

```diff
@@ -1,11 +1,14 @@
 import * as path from 'node:path';
 import * as vite from 'vite';
 import { unstable_getMiniflareWorkerOptions } from 'wrangler';
+import type { AssetConfig } from './assets';
 import type { SourcelessWorkerOptions } from 'wrangler';
 
 export interface WorkerOptions {
 	main: string;
 	wranglerConfig?: string;
+	// TODO: tighten up types so assets can only be bound to entry worker
```

Comment:
I'm not sure this should be a goal. Assets should be configurable per Worker.

Reply:
I don't disagree that the longer term goal should be for assets to be configurable per worker but I think that, even then, the entry worker may still be distinct in the way it needs to be configured. Again, that needs to be part of broader discussions about the API and I feel it's outside the scope of this PR.